packages/ui/src/views/Edit/index.tsx

```diff
@@ -286,7 +286,7 @@ export const DefaultEditView: React.FC<ClientSideEditViewProps> = ({
         const { state } = await getFormState({
           id,
           collectionSlug,
-          data: json?.doc,
+          data: json?.doc || json?.result,
```

Comment:
Can we investigate why this response is different across collections vs globals?

Reply:
Going to release this fix asap and then will review for why the return here might be different